aws/assume-role/README.md

```diff
@@ -55,10 +55,57 @@ tasks:
     call: aws/install-cli 1.0.0
   - key: assume-role
     use: aws-cli
-    call: aws/assume-role 1.0.2
+    call: aws/assume-role 1.1.0
     with:
       oidc-token: ${{ vaults.your-vault.oidc.your-token }}
       region: us-east-2
       role-to-assume: arn:aws:iam::your-account-id:role/your-role
       profile-name: your-profile
 ```
+
+To assume another role, via chaining:
+
+```yaml
+tasks:
+  - key: aws-cli
+    call: aws/install-cli 1.0.0
```

Comment:
If you don't mind pushing a quick update, I've started inserting newlines between tasks in docs for readability.

Reply:
Smart!